Statement parser doesn't handle non-ASCII characters well

org.skife.jdbi.v2.exceptions.CallbackFailedException: org.skife.jdbi.v2.exceptions.UnableToCreateStatementException: Exception parsing for named parameter replacement [statement:"INSERT INTO 营业日数据(餐厅编号,餐厅名称,业务日期,节假日,时段,一级报表项目,二级报表项目,是否末级,金额,标识)  VALUES('0101','福建厦门松柏餐厅','2016-04-29',null,'休闲','DI',null,'是',768.5,'0101-2016.4')", located:"INSERT INTO 营业日数据(餐厅编号,餐厅名称,业务日期,节假日,时段,一级报表项目,二级报表项目,是否末级,金额,标识)  VALUES('0101','福建厦门松柏餐厅','2016-04-29',null,'休闲','DI',null,'是',768.5,'0101-2016.4')", rewritten:"null", arguments:{ positional:{}, named:{}, finder:[]}]
    at org.skife.jdbi.v2.DBI.withHandle(DBI.java:301)
    at com.ei.db.dao.DefaultDao.executeUpdate(DefaultDao.java:584)
    at hkl.fbi.service.YyrsjService.yyrsj2(YyrsjService.java:34)
    at hkl.fbi.service.YyrsjService.yyrsj(YyrsjService.java:104)
    at hkl.fbi.service.YyrsjService.main(YyrsjService.java:116)
Caused by: org.skife.jdbi.v2.exceptions.UnableToCreateStatementException: Exception parsing for named parameter replacement [statement:"INSERT INTO 营业日数据(餐厅编号,餐厅名称,业务日期,节假日,时段,一级报表项目,二级报表项目,是否末级,金额,标识)  VALUES('0101','福建厦门松柏餐厅','2016-04-29',null,'休闲','DI',null,'是',768.5,'0101-2016.4')", located:"INSERT INTO 营业日数据(餐厅编号,餐厅名称,业务日期,节假日,时段,一级报表项目,二级报表项目,是否末级,金额,标识)  VALUES('0101','福建厦门松柏餐厅','2016-04-29',null,'休闲','DI',null,'是',768.5,'0101-2016.4')", rewritten:"null", arguments:{ positional:{}, named:{}, finder:[]}]
    at org.skife.jdbi.v2.ColonPrefixNamedParamStatementRewriter.rewrite(ColonPrefixNamedParamStatementRewriter.java:69)
    at org.skife.jdbi.v2.SQLStatement.internalExecute(SQLStatement.java:1296)
    at org.skife.jdbi.v2.Update.execute(Update.java:56)
    at org.skife.jdbi.v2.BasicHandle.update(BasicHandle.java:297)
    at com.ei.db.dao.DefaultDao.lambda$5(DefaultDao.java:585)
    at org.skife.jdbi.v2.DBI.withHandle(DBI.java:298)
    ... 4 more
Caused by: java.lang.IllegalArgumentException: NoViableAltException('营'@[])
    at org.skife.jdbi.rewriter.colon.ColonStatementLexer.reportError(ColonStatementLexer.java:26)
    at org.skife.jdbi.org.antlr.runtime.Lexer.nextToken(Lexer.java:107)
    at org.skife.jdbi.v2.ColonPrefixNamedParamStatementRewriter.parseString(ColonPrefixNamedParamStatementRewriter.java:106)
    at org.skife.jdbi.v2.ColonPrefixNamedParamStatementRewriter.rewrite(ColonPrefixNamedParamStatementRewriter.java:65)
    ... 9 more
Caused by: NoViableAltException('营'@[])
    at org.skife.jdbi.rewriter.colon.ColonStatementLexer.mTokens(ColonStatementLexer.java:872)
    at org.skife.jdbi.org.antlr.runtime.Lexer.nextToken(Lexer.java:89)
    ... 11 more

--- a/core/src/test/java/org/jdbi/v3/core/TestHandle.java
+++ b/core/src/test/java/org/jdbi/v3/core/TestHandle.java
@@ -59,4 +59,10 @@ public class TestHandle
         h.close();
         assertThat(h.isClosed()).isTrue();
     }
+
+    @Test
+    public void testMrWinter() {
+        final Handle h = dbRule.getSharedHandle();
+        h.execute("CREATE TABLE \"\u2603\" (pk int primary key)");
+    }
 }

--- a/docs/src/adoc/index.adoc
+++ b/docs/src/adoc/index.adoc
@@ -4884,6 +4884,9 @@ handle.createUpdate("insert into characters (id, name) values (#id, #name)")
       .execute();
 ----
 
+[NOTE]
+The default parsers do not readily recognize non-Latin characters as identifiers;
+they must be surrounded by double quotes.
 
 For you fearless adventurers who have read the
 link:https://www.amazon.com/Compilers-Principles-Techniques-Tools-2nd/dp/0321486811[Dragon book^],

